### ISSUE TITLE
Plugin crashing Homebridge  w/ Wyze v2 camera

### ISSUE THREAD

**Issue by tbelcher** [seydx/homebridge-camera-ui#366]
**Describe the bug**
Homebridge ends up crashing after attempting to start with the old camera I've added.

**To Reproduce**
Install plugin, add Wyze camera and restart Homebridge

**Expected behavior**
Able to add Wyze v2 camera and view stream in the Home app

**Logs**
Jan 17 22:08:13 raspberrypi homebridge[22882]: 2022-01-18T06:08:13.335Z socket.io-parser encoding packet {"type":2,"data":["logMessage","\u001b[37m[1/17/2022, 10:08:09 PM]\u001b[39m \u001b[36m[CameraUI]\u001b[39m \u001b[37mWyze v2: Setting up camera, please be patient...\u001b[39m"],"nsp":"/"}

Finally crashing with:
Jan 17 22:08:13 raspberrypi homebridge[22882]: node:internal/console/constructor:290
Jan 17 22:08:13 raspberrypi homebridge[22882]:           throw e;
Jan 17 22:08:13 raspberrypi homebridge[22882]:           ^
Jan 17 22:08:13 raspberrypi homebridge[22882]: RangeError: Maximum call stack size exceeded
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at String.replace (<anonymous>)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at debug (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/node_modules/debug/src/common.js:91:22)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Encoder.encode (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/dist/index.js:35:9)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Adapter.broadcast (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-adapter/dist/index.js:107:45)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at BroadcastOperator.emit (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/broadcast-operator.js:109:22)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Namespace.emit (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/namespace.js:170:73)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Server.<computed> [as emit] (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/index.js:576:33)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Object.file (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/camera.ui/src/services/logger/logger.service.js:241:27)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at /usr/local/lib/node_modules/homebridge-camera-ui/services/logger/logger.service.js:127:84
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Socket.write (/usr/local/lib/node_modules/homebridge-camera-ui/services/logger/logger.service.js:72:9)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Function.log (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/node_modules/debug/src/node.js:194:24)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at debug (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/node_modules/debug/src/common.js:113:10)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Encoder.encode (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/dist/index.js:35:9)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Adapter.broadcast (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-adapter/dist/index.js:107:45)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at BroadcastOperator.emit (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/broadcast-operator.js:109:22)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Namespace.emit (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/namespace.js:170:73)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Server.<computed> [as emit] (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/index.js:576:33)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Object.file (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/camera.ui/src/services/logger/logger.service.js:241:27)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at /usr/local/lib/node_modules/homebridge-camera-ui/services/logger/logger.service.js:127:84
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Socket.write (/usr/local/lib/node_modules/homebridge-camera-ui/services/logger/logger.service.js:72:9)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Function.log (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/node_modules/debug/src/node.js:194:24)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at debug (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/node_modules/debug/src/common.js:113:10)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Encoder.encode (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/dist/index.js:35:9)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Adapter.broadcast (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-adapter/dist/index.js:107:45)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at BroadcastOperator.emit (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/broadcast-operator.js:109:22)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Namespace.emit (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/namespace.js:170:73)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Server.<computed> [as emit] (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/index.js:576:33)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Object.file (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/camera.ui/src/services/logger/logger.service.js:241:27)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at /usr/local/lib/node_modules/homebridge-camera-ui/services/logger/logger.service.js:127:84
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Socket.write (/usr/local/lib/node_modules/homebridge-camera-ui/services/logger/logger.service.js:72:9)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Function.log (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/node_modules/debug/src/node.js:194:24)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at debug (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/node_modules/debug/src/common.js:113:10)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Encoder.encode (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-parser/dist/index.js:35:9)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Adapter.broadcast (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io-adapter/dist/index.js:107:45)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at BroadcastOperator.emit (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/broadcast-operator.js:109:22)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Namespace.emit (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/namespace.js:170:73)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Server.<computed> [as emit] (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/socket.io/dist/index.js:576:33)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Object.file (/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/camera.ui/src/services/logger/logger.service.js:241:27)
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at /usr/local/lib/node_modules/homebridge-camera-ui/services/logger/logger.service.js:127:84
Jan 17 22:08:13 raspberrypi homebridge[22882]:     at Socket.write (/usr/local/lib/node_modules/homebridge-camera-ui/services/logger/logger.service.js:72:9)

**Environment**
- Node.js Version: v16.13.0.
- Homebridge Version: v1.4.0.-beta 7
- Camera UI Version: v5.0.9
- Operating System: raspberry pi Buster
- Browser: XXX

**Comment by seydx** [comment]
@tbelcher 

can you post your config.json pls

**Comment by tbelcher** [comment]
{
            "name": "CameraUI",
            "port": 8081,
            "debug": false,
            "mqtt": {
                "active": false
            },
            "http": {
                "active": false
            },
            "smtp": {
                "active": false
            },
            "ftp": {
                "active": false
            },
            "ssl": {
                "active": false
            },
            "cameras": [
                {
                    "name": "Wyze v2",
                    "manufacturer": "wye",
                    "serialNumber": "none",
                    "motion": true,
                    "unbridge": true,
                    "hsv": true,
                    "prebuffering": true,
                    "prebufferLength": 4,
                    "videoConfig": {
                        "source": "-i rtsp://#####:#####@192.168.86.59/live",
                        "rtspTransport": "tcp",
                        "maxStreams": 2,
                        "maxWidth": 0,
                        "maxHeight": 0,
                        "vcodec": "copy",
                        "acodec": "libfdk_aac",
                        "stimeout": 10,
                        "audio": false
                    },
                    "videoanalysis": {
                        "active": true
                    }
                }
            ],
            "platform": "CameraUI"
        }

**Comment by seydx** [comment]
hey, can you try v5.0.10-beta.5 pls

**Comment by tbelcher** [comment]
Installed v5.0.10-beta.5 and the behavior is the same.  Even though I've removed and re-installed it a few times, I am not able to get to the "interface" tab.  Cameras and Config both seem to work, but clicking on interface gives me a host can't be reached error.  Seems like the web portion isn't running.

I've enabled DEBUG in the config, but I'm not seeing any additional output.  Is there somewhere else I should look?

Thank you
